lib/processors/bundlers/moduleBundler.js

```diff
@@ -135,6 +135,7 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
  * @param {string} [parameters.options.targetUi5CoreVersion] Optional semver compliant sap.ui.core project version, e.g '2.0.0'.
 				This allows the bundler to make assumptions on available runtime APIs.
 				Omit if the ultimate UI5 version at runtime is unknown or can't be determined.
+ * @param {@ui5/project/build/helpers/TaskUtil|object} [parameters.taskUtil] TaskUtil
```

Comment:
obsolete parameter? or was it already in use but undocumented?

Reply:
I have removed it in https://github.com/SAP/ui5-builder/pull/1036/files#diff-255002459f4f366a0dceae0db4927dd06d44894f887b3ba210884d9a399f48d1L138